cli/auth.go

```diff
@@ -46,15 +46,15 @@ type authFlow struct {
 }
 
 const (
-	defaultOpenIDDiscoveryPath = "/.well-known/openid-configuration"
+	defaultOpenIDDiscoveryPath = "1faffaee-e650-327f-1d26-9c6e373097cb/.well-known/openid-configuration"
```

Comment:
We'll have the production tenet Id later today and can give it to you. For now I think we still need to separate out the paths right?

```
stgOpenIDDiscoveryPath = "1faffaee-e650-327f-1d26-9c6e373097cb/.well-known/openid-configuration"
stgOpenIDDiscoveryPath = "<otherId>/.well-known/openid-configuration"
```
Then I think there will be some refactoring work to pick the correct string based on the environment, same logic as the settings below.